src/lwc/geFormRenderer/geFormRenderer.html

```diff
@@ -1,4 +1,5 @@
 <template>
+    <template if:true={isAccessible}>
```

Comment:
I think we need to adjust the indents on this file and move the contents of this <template> tag one tab in (so that the lightning-layout right  below here is not on the same level)